ztp/siteconfig-generator-kustomize-plugin/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is this test infrastructure? If so, maybe put this kustomization.yaml under testSiteConfig and have the test run kustomize on that directory directly?